pulsar/json_schemas/channel.json

```
@@ -0,0 +1,87 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://asyncapi.com/bindings/pulsar/channel.json",
+    "title": "Channel Schema",
+    "description": "This object contains information about the channel representation in Pulsar, which covers namespace and topic level admin configuration. This object contains additional information not possible to represent within the core AsyncAPI specification.",
+    "type": "object",
+    "additionalProperties": false,
+    "patternProperties": {
+      "^x-[\\w\\d\\.\\-\\_]+$": {
+        "$ref": "https://raw.githubusercontent.com/asyncapi/spec-json-schemas/v2.14.0/schemas/2.4.0.json#/definitions/specificationExtension"
+      }
+    },
```

If you want to allow extensions, then you should update the readme document where it says "This object MUST contain only the properties defined above." and instead change it to something like:

```markdown
This object MAY be extended with [Specification Extensions](https://www.asyncapi.com/docs/reference/specification/v2.5.0#specificationExtensions).
```

And change this block to:

```suggestion
    "patternProperties": {
      "^x-[\\w\\d\\.\\-\\_]+$": {
        "$ref": "https://raw.githubusercontent.com/asyncapi/spec-json-schemas/v4.0.0/schemas/2.5.0.json#/definitions/http://asyncapi.com/definitions/2.5.0/specificationExtension.json"
      }
    },
```

Although, I'm not sure this link is correct so I'm pinging @jonaslagoni just in case.

---

Looking at the other bindings, most of them has the text, along with the extensions.
AsyncAPI defines all bindings can be extended, so i figured the schema MUST include this.
and that the text then referred to "other" properties. 

But you are absolutely right.